### ISSUE TITLE
Hypersensitivity to Thermal Stimulus (Three Terms Requested)

### ISSUE THREAD

**Issue by avaccunningham** [obophenotype/human-phenotype-ontology#10574]
**Preferred term label:**
Hypersensitivity to thermal stimulus

**Synonyms**
Thermal hypersensitivity

**Definition (free text, please give PubMed ID)**
A decreased tolerance to a thermal stimulus.

**Parent term (use hpo.jax.org/app)**
Sensory hypersensitivity

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Dental pain

**Your nano-attribution (ORCID)**
https://orcid.org/0009-0003-5583-4731

**Synonyms**
Cold hypersensitivity

**Definition (free text, please give PubMed ID)**
A decreased tolerance to a cold stimulus.

**Parent term (use hpo.jax.org/app)**
Thermal hypersensitivity

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Dental pain

**Your nano-attribution (ORCID)**
https://orcid.org/0009-0003-5583-4731

**Synonyms**
Hot hypersensitivity

**Definition (free text, please give PubMed ID)**
A decreased tolerance to a hot stimulus.

**Parent term (use hpo.jax.org/app)**
Thermal hypersensitivity

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Dental pain

**Your nano-attribution (ORCID)**
https://orcid.org/0009-0003-5583-4731

@addiehl

**Comment by pnrobinson** [comment]
@avaccunningham 
Can we be more specific? Are these terms talking about sensitivity of a tooth that already has a tooth ache? 
The HPO is about abnormal phenotypes, and having a diseased tooth be sensitive to hot or cold is probably normal for the circumstances and it amy not add much above and beyond annotating to Tooth pain. Can you provide more context?

**Comment by pnrobinson** [comment]
I will close this for now, please reopen if needed for futher discussion.
